### DISCUSSION TITLE
Faderfox EC4 Feedback Script (Request)

### DISCUSSION THREAD

**Discussion by dna598** [helgoboss/helgobox#930] [Ideas]
Could anyone that is suitably qualified kindly have a look at implementing the Faderfox  EC4 feedback script please? 
V2 firmware has been out for quite a while now, and this excellent controller really deserves support in Realearn. 
Thanks!

Docs attached.

[EC4 sysex documentation.zip](https://github.com/user-attachments/files/15801559/EC4.sysex.documentation.zip)

**Comment by helgoboss** [comment]
Supporting the EC4 displays is a bit of effort and I'm not particularly interested in this myself at the moment. Funding can help though: https://forum.cockos.com/showthread.php?t=178015&page=115&p=2817125

**Reply by dna598** [reply]
Sure thing. Would perusing the documentation in Mossgrabers's implementation help? Just a thought- I have no idea about this stuff.

https://github.com/git-moss/DrivenByMoss/blob/master/resources/Faderfox/EC4_DrivenByMoss_Setup.syx

**Comment by helgoboss** [comment]
I added something here: https://github.com/helgoboss/helgobox/wiki/Faderfox-EC4
